mono/metadata/icall.c

```diff
@@ -8425,3 +8425,26 @@ mono_register_jit_icall (gconstpointer func, const char *name, MonoMethodSignatu
 	return mono_register_jit_icall_full (func, name, sig, no_wrapper, NULL);
 }
 
+ICALL_EXPORT int
```

Comment:
One wonders if this can overflow.

Reply:
It follows MSDN though https://msdn.microsoft.com/en-us/library/system.gc.collectioncount(v=vs.110).aspx.